.github/workflows/main.yaml

```diff
@@ -0,0 +1,129 @@
+name: main
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
+env:
+  go-version: 1.15
+jobs:
+  build:
+    name: Build CKE
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Check static resources
+        run: |
+            cp ./static/resources.go /tmp/resources.go
+            make static
+            cmp /tmp/resources.go ./static/resources.go
+      - run: docker run -d --network host quay.io/coreos/etcd:v3.3
+      - run: make test-tools
+      - run: make test GOBIN=$(pwd)/docker
+      - name: build-image
+        run: docker build -t quay.io/cybozu/cke:latest ./docker
+  mtest:
+    name: Mtest
+    runs-on: ubuntu-20.04
+    needs: build
+    strategy:
+      matrix:
+        suite: [functions, robustness, operators]
+    env:
+      INSTANCE_NAME: cke-${{ matrix.suite }}-${{ github.run_number }}
+      SUITE: ${{ matrix.suite }}
+      CLUSTER: "cke-cluster.yml"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Set up Cloud SDK for cke
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.NECO_TEST_PROJECT_ID }}
+          service_account_key: ${{ secrets.NECO_TEST_SERVICE_ACCOUNT }}
+          export_default_credentials: true
+      - name: Watch cke logs
+        run: |
+            if [ -f .skip ]; then exit 0; fi
+            ./bin/watch_service cke &
```

Comment:
GitHub actions does not allow us to use a background run step, thus the logs are invisible after the step is finished.
We need another run step to display the log.
https://github.com/cybozu-go/moco/blob/main/.github/workflows/ci.yaml#L66

Reply:
Thank you. I'll fix this